src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -0,0 +1,66 @@
+package cromwell.engine
+
+import java.util.{Calendar, UUID}
+
+import akka.testkit.TestActorRef
+import cromwell.binding.values.WdlString
+import cromwell.engine.db.{QueryWorkflowExecutionResult, DummyDataAccess}
+import cromwell.engine.workflow.WorkflowManagerActor
+import cromwell.engine.workflow.WorkflowManagerActor.{SubmitWorkflow, WorkflowOutputs, WorkflowStatus}
+import cromwell.util.SampleWdl
+import cromwell.util.SampleWdl.HelloWorld
+import cromwell.{CromwellTestkitSpec, binding}
+
+
+class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActorSpec") {
```

Comment:
Minor mergetastrophe: `WorkflowManagerActorSpec` is here under its current name as well as its previous name `ActorWorkflowManagerSpec`.